### PR TITLE
Upgrade @alephium/web3 to 0.2.0-rc.33

### PR DESCRIPTION
**Pull request by h0ngcha0**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,7 +23,7 @@
     "format": "prettier --loglevel warn --write \"**/*.{js,jsx,ts,tsx,css,md,yml,json}\"",
     "dev": "lerna run dev --parallel",
     "dev:ui": "lerna run dev:ui --parallel",
-    "clean": "rm -rf packages/extension/dist packages/get-extension-wallet/dist",
+    "clean": "rm -rf packages/extension/dist packages/get-extension-wallet/dist packages/dapp/dist && rm -rf packages/extension/node_modules packages/get-extension-wallet/node_modules packages/dapp/node_modules",
```
Comment (nop33) [inline package.json:26]: Is there a need to repeat the `rm -rf` command?

```suggestion
    "clean": "rm -rf packages/extension/dist packages/get-extension-wallet/dist packages/dapp/dist packages/extension/node_modules packages/get-extension-wallet/node_modules packages/dapp/node_modules",
```

**Review by nop33 — COMMENTED** [review]
Nice job @h0ngcha0!

I noticed that your changes do not respect the formatting rules defined by the `.prettierrc` config file. Personally, I have set up my editor to consult this file every time I save a file and make the appropriate changes. Maybe you could do something similar so that you don't have to run `yarn format` all the time!

PS: I noticed that `yarn format` messes up some type imports, we'll need to have a look into that.

PS2: Also, have a look at my #13 PR and merge it before merging this one :)

File: package.json (hunk 1 above)
Comment (h0ngcha0) [reply package.json:26]: Nice, will update, thanks!

**Comment by h0ngcha0** [comment]
Thanks for the CR!

> Personally, I have set up my editor to consult this file every time I save a file and make the appropriate changes. Maybe you could do something similar so that you don't have to run yarn format all the time!

I will look into that :+1:

**Pull request merged**